tests/legacy-cli/e2e/tests/i18n/ivy-localize-dl.ts

```diff
@@ -14,27 +14,21 @@ export default async function() {
     config.angularCompilerOptions.disableTypeScriptVersionCheck = true;
   });
 
-  // TODO: re-enable all locales once localeData support is added.
-  const tempLangTranslations = langTranslations.filter(l => l.lang == 'fr');
-
   // Build each locale and verify the output.
-  // NOTE: this should not fail in general, but multi-locale translation is currently disabled.
-  // TODO: remove expectToFail once localeData support is added.
-  await expectToFail(() => ng('build', '--localize', 'true'));
   await ng('build');
-  for (const { lang, outputPath, translation } of tempLangTranslations) {
+  for (const { lang, outputPath, translation } of langTranslations) {
     await expectFileToMatch(`${outputPath}/main-es5.js`, translation.helloPartial);
     await expectFileToMatch(`${outputPath}/main-es2015.js`, translation.helloPartial);
     await expectToFail(() => expectFileToMatch(`${outputPath}/main-es5.js`, '$localize`'));
     await expectToFail(() => expectFileToMatch(`${outputPath}/main-es2015.js`, '$localize`'));
 
     // Verify the locale ID is present
-    await expectFileToMatch(`${outputPath}/main-es5.js`, lang);
-    await expectFileToMatch(`${outputPath}/main-es2015.js`, lang);
+    await expectFileToMatch(`${outputPath}/vendor-es5.js`, lang);
```

Comment:
How come the locale is written to `vendor` rather than `main`?

Reply:
It's written to the same location to that of the Angular FW code.  The locale data is also placed there for builds since it is technically vendor code.
For an actual production build all of it would be in main since there is no vendor bundle.